cleanup - purge office365 emails with slack (#125)
rapid7/insightconnect-workflows: workflows/Purge_Office_365_Emails_with_Slack/help.md

# Description

> Understanding the scope of a phishing campaign is critical to ensuring no one in your organization takes the bait. This workflow runs a search from a command in Slack and purges emails across Office365 inboxes.

Reviewer: ```suggestion
Understanding the scope of a phishing campaign is critical to ensuring no one in your organization takes the bait. This workflow runs a search from a command in Slack and purges emails across Microsoft Office 365 inboxes.
```

Sample Slack Trigger Commands:

> `!purge-email subject="A phishy email"`

Reviewer 2: ```suggestion
`@Slackbot !purge-email subject="A phishy email"`
```

> `!purge-email subject="A phishy email" from="example.com" body="Click here for free stuff" `

Reviewer 2: ```suggestion
`@Slackbot !purge-email subject="A phishy email" from="example.com" body="Click here for free stuff" `
```

> `@Slackbot !purge-email subject="A phishy email" from="example.com" body="Click here for free stuff" `

Reviewer 2: ```suggestion
`@Slackbot !purge-email subject="A phishy email" from="example.com" body="Click here for free stuff"`
```

> `!purge-email subject="A phishy email" delete=true`

Reviewer 2: ```suggestion
`@Slackbot !purge-email subject="A phishy email" delete=true`
```
